.github/workflows/quality.yaml

```diff
@@ -45,6 +45,9 @@ jobs:
       - name: Clippy (default features + gdb)
         run: cargo clippy --locked --all --all-targets --tests --features "gdb" -- -D warnings
 
+      - name: Clippy (default features + guest_debug)
+        run: cargo clippy --locked --all --all-targets --tests --features "guest_debug" -- -D warnings
```

Comment:
Ah yes it was added to build.yaml but I forgot it's also needed in quality.yaml.